src/Okta.Idx.Sdk.E2ETests/Helpers/TestUserHelper.cs

```diff
@@ -38,15 +40,59 @@ public async Task<TestUserProperties> GetActivePasswordUserAsync()
             };
         }
 
-        public TestUserProperties GetUnassignedUser()
+        public async Task<TestUserProperties> GetUnassignedUser()
         {
+            await CleanUpAsync();
+
+            var oktaUser = await _oktaHelper.CreateUnassignedUserIdentifiedWithPasswordAsync(_a18nProfile.EmailAddress, _configuration.UserPassword);
```

Comment:
hmm `CreateUnassignedUserIdentifiedWithPasswordAsync` is not implemented, what am I missing?

Reply:
It's not for Feature 3, I started to implement next feature scenarios here.